components/toast/style.css

```diff
@@ -0,0 +1,117 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.guic-toast-container{
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    overflow-y: hidden;
+    overflow-x: visible;
+    width: 300px;
+}
+
+.guic-toast-top{
+    top: 15px;
+}
+
+.guic-toast-bottom{
+    bottom: 15px;
+}
+
+.guic-toast-right{
+    right: 15px;
+}
+
+.guic-toast-left{
+    left: 15px;
+}
+
+.guic-toast-center{
+    left: 50%;
+    transform: translateX(-50%);
+}
+
+.guic-toast-bottom gameface-toast{
+    animation: guic-toast-slide-up 1s ease-in-out;
+    margin-top: 5px;
+}
+
+.guic-toast-top gameface-toast{
+    animation: guic-toast-slide-down 1s ease-in-out;
+    margin-bottom: 5px;
+}
+
+@keyframes guic-toast-slide-up {
+    0% {
+        transform: translateY(100%);
+        opacity: 0;
+    }
+    100% {
+        transform: translateY(0%);
+        opacity: 1;
+    }
+}
+
+@keyframes guic-toast-slide-down {
+    0% {
+        transform: translateY(-100%);
+        opacity: 0;
+    }
+    100% {
+        transform: translateY(0%);
+        opacity: 1;
+    }
+}
+
+gameface-toast{
+    position: absolute;
+    visibility: hidden;
+}
+
+gameface-toast.guic-toast-show{
+    position: relative;
+    visibility: visible;
+}
+
+@keyframes  guic-toast-fade-out {
+    from {
+        opacity: 1;
+    }
+    to {
+        opacity: 0;
+    }
+}
+
+gameface-toast.guic-toast-hide {
+    animation: guic-toast-fade-out 0.5s ease forwards;
+}
+
+.guic-toast-container gameface-toast{
```

Comment:
Same as above

Reply:
If I don't use a selector for the hide logic with a specificity of at least (0,1,1) the selectors: 
`.guic-toast-top gameface-toast` and `.guic-toast-bottom gameface-toast` don't get overwritten and the toast doesn't hide.

Reply:
Fixed